packages/main/src/RadioButton.js

```diff
@@ -284,7 +284,7 @@ class RadioButton extends UI5Element {
 		}
 	}
 
-	onclick() {
+	_onClick() {
```

Comment:
As I already mention in the ui5-label refactoring pull request (you can see it for more details), we decided to go for the following naming: "_onclick" "_onkeydown" "_onkeyup"

And for the commit message (it is now just "Radio Button", perhaps the last commit has overwritten you initial commit message), keep in mind that we start with a verb and use imperative form:
"refactor(ui5-radiobutton): subscribe event handlers within HBS template"

Reply:
All comments were applied